crates/epaint/src/text/fonts.rs

```diff
@@ -878,7 +878,7 @@ impl GalleyCache {
             // * https://github.com/emilk/egui/issues/5084
             // * https://github.com/emilk/egui/issues/5163
 
-            job.wrap.max_width = job.wrap.max_width.round();
+            Arc::make_mut(&mut job).wrap.max_width = job.wrap.max_width.round();
```

Comment:
This is slightly unfortunate and could be avoided by threading a separate `max_width` throughout layout but that sounds like a lot of work while when working with a lot of text you probably don't necessarily need line wrapping anyway.